examples_tests/13.BRDF_Explorer/BRDF.h

```diff
@@ -48,6 +48,8 @@ class BRDF {
         BRDF(video::IVideoDriver* driver);
```

Comment:
ok you need to rename this object to `BRDFExplorerApp` or something similar, simply calling it `BRDF` is really confusing

Reply:
Refactored.